libr/asm/arch/arc/gnu/arcompact-dis.c

```diff
@@ -558,7 +558,7 @@ write_instr_name_(struct arcDisState *state,
     }
   if (flag) strcat(state->instrBuffer, ".f");
   if (state->nullifyMode)
-    if (strstr(state->instrBuffer, ".d") == NULL)
+    if (!strstr(state->instrBuffer, ".d"))
```

Comment:
also reverting changes in `libr/asm/arch/xtensa/gnu/elf32-xtensa.c` too in that case